src/round.rs

```diff
@@ -176,4 +324,113 @@ mod tests {
         assert_eq!(dt.trunc_subsecs(0).nanosecond(), 1_000_000_000);
         assert_eq!(dt.trunc_subsecs(0).second(), 59);
     }
+
+    #[test]
+    fn test_duration_round() {
+        let dt = Utc.ymd(2016, 12, 31).and_hms_nano(23, 59, 59, 1_75_500_000);
```

Comment:
`1_75_500_000` is a bit of a weird spacing? Is it intentional?

Reply:
whoops, no, typo